share/locale/musescore_pt.ts

```diff
@@ -7365,7 +7365,7 @@ text x-height):</source>
     <message>
         <location filename="../../src/notation/internal/notationuiactions.cpp" line="71"/>
         <source>Fret %1 (TAB)</source>
-        <translation>Traste nº1 (TAB)</translation>
+        <translation>Traste %1 (TAB)</translation>
```

Comment:
Could it be that they mean `Traste nº%1 (TAB)`?

Reply:
Possible, added